.ci/setup.sh

```diff
@@ -48,8 +48,10 @@ else  # Linux
     fi
 fi
 
-if [[ $TRAVIS == "true" ]] || [[ $GITHUB_ACTIONS == "true" ]] || [[ $OS_NAME == "macos" ]]; then
-    sh conda.sh -b -p $CONDA
+if [[ "${TASK:0:9}" != "r-package" ]]; then
```

Comment:
@StrikerRUS I was torn between doing this `!=` thing of just adding  this:

```shell
if [[ "${TASK:0:9}" != "r-package" ]]; then
    exit 0
fi
```

Do you have a strong preference?

Reply:
@jameslamb 

> or just adding this before the conda steps:

I think the second way will be more readable.

Reply:
Never mind! I changed my mind 😄

Reply:
haha ok. Easy to change if you ever want to.